docs/installation.md

```diff
@@ -41,8 +41,9 @@ you log into its account:
 
 ```shell
 $ virtualenv --python=python2.7 $HOME
-$ source bin/activate  
+$ . bin/activate  
```

Comment:
Out of curiosity, what does the `dot` approach changes to the `source` one?

Reply:
`.` is available in any `shell` including `sh`, `ash`... whereas `source` is only available in `bash`.